packages/google_identity_services_web/README.md

```diff
@@ -29,7 +29,7 @@ script tag [as recommended](https://developers.google.com/identity/gsi/web/guide
 Place the `script` tag in the `<head>` of your site, next to the script tag that
 loads `flutter.js`, so the browser can downloaded both in parallel:
 
-<?code-excerpt "../../web/index-with-script-tag.html (script-tag)"?>
```

Comment:
i mostly tried to do update mechanically, rather than optimize them, because when i changed them i was still trying to just prove to myself that the output wouldn't change. i'm very happy to study these more carefully and make them more useful, i'll do that shortly.

Reply:
i updated the two cases where we used `../` to have more reasonable path bases.